backend/YouTubeV2.Api.Tests/VideoControllerTests/AddVideoMetadataAsyncTests.cs

```diff
@@ -42,8 +40,7 @@ public async Task Initialize()
         {
             _userServiceMock.Setup(x => x.GetByIdAsync(It.IsAny<string>())).ReturnsAsync(_user);
             _dateTimeProviderMock.Setup(x => x.UtcNow).Returns(_utcNow);
-            _blobImageService.Setup(x => x.UploadVideoThumbnailAsync(It.IsAny<string>(), It.IsAny<string>(), It.IsAny<CancellationToken>()));
```

Comment:
these changes are not connected to dtos and should be reverted

Reply:
i didn't touch this file, reverting